ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/AffectedImagesTable.tsx

```diff
@@ -176,7 +177,13 @@ function AffectedImagesTable({
                                             : `${imageComponents.length} components`}
                                     </Td>
                                     <Td dataLabel="First discovered">
-                                        <DateDistance date={scanTime} />
+                                        {isNonEmptyArray(vulnerabilities) ? (
+                                            <DateDistance
+                                                date={getEarliestDiscoveredAtTime(vulnerabilities)}
+                                            />
+                                        ) : (
+                                            'N/A'
```

Comment:
You might ask in channel about pro and con of **N/A** versus explicit phrase.

For NVD CVSS, explicit guidance to render **Not available** because **Not applicable** is incorrect in that context. Especially if one is clearer than the other here also, worthwhile to share on Wednesday and Friday so everyone knows to confirm meanings in context.

Reply:
Agreed that "Not available" is correct in this context, since this is an edge case where the data is simply missing and should not in general occur.